src/coffee/directives/search-box.coffee

```diff
@@ -38,6 +38,9 @@ angular.module('uiGmapgoogle-maps')
       
       link: (scope, element, attrs, mapCtrl) =>
         GoogleMapApi.then (maps) =>
+          unless scope.template?
```

Comment:
this works , but would it not be better to watch scope.template until it is defined?

```
d = $q.defer()
promise = d.promise
unbind = scope.watch 'template', (newVal) ->
    d.resolve()

promise.then ->
   $http.get(scope.template, { cache: $templateCache })
             .success (template) =>
               if angular.isUndefined scope.events
```